migrate-tags.js

```diff
@@ -76,23 +75,20 @@ async function addTrackTag(client, rId, trackId, tag) {
     trackId,
     tag.taggerId
   ];
-  await client.query(text, values).catch(e => logger.error(e.stack));
+  await client.query(text, values).catch(e => {
+    logger.error(e.stack);
+    process.exit(0);
```

Comment:
this is fine but you could have also just let the exception bubble up with largely the same result (i.e. no `catch` at all)